IUser.cs

```
@@ -0,0 +1,28 @@
+﻿using System;
+
+public interface IUser
+{
```

There should be methods responsible for retreiving a user from the database. Probably as a constructor. 
Should be implemented in the interface, for the separate users, you can simply throw a NotImplemented exception.